javascript-if/main.js

```diff
@@ -0,0 +1,108 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+// function isUnderFive(number) {
+//   if (number < 5) {
+//     return true;
+//   } else {
+//     return false;
+//   }
+// }
+
+function isUnderFive(number) {
+  return number < 5;
+}
+
+// console.log('value of isUnderFive: ', isUnderFive(4));
+// console.log('value of isUnderFive: ', isUnderFive(10));
+// console.log('value of isUnderFive: ', isUnderFive(5));
```

Comment:
It would greatly help the readability of your `log`s if you also included the value of the argument.